larky/src/main/java/com/verygood/security/larky/ModuleSupplier.java

```diff
@@ -100,7 +101,8 @@ public class ModuleSupplier {
       NetworkTokenModule.INSTANCE,
       CerebroModule.INSTANCE,
       ChaseModule.INSTANCE,
```

Comment:
can these be alphabetized?

Reply:
resolving this, as the list has been alphabetized.